libimage/filters.go

```diff
@@ -95,9 +95,12 @@ func (r *Runtime) compileImageFilters(ctx context.Context, options *ListImagesOp
 	for _, f := range options.Filters {
 		var key, value string
 		var filter filterFunc
-		split := strings.SplitN(f, "=", 2)
+		split := strings.SplitN(f, "!=", 2)
 		if len(split) != 2 {
```

Comment:
Could you do a positive test here and set a boolean negative=true
negativeFilter=false
if len(split) == 2 {
negativeFilter=true
} else {
...
}

Reply:
Thanks, that seems faster indeed. Changed to this suggestion